raiden-ts/src/config.ts

```diff
@@ -123,6 +123,15 @@ export function makeDefaultConfig(
       ? 'https://raw.githubusercontent.com/raiden-network/raiden-service-bundle/master/known_servers/known_servers-production-v1.2.0.json'
       : 'https://raw.githubusercontent.com/raiden-network/raiden-service-bundle/master/known_servers/known_servers-development-v1.2.0.json';
 
+  // SVT also uses 18 decimals, like Ether, so parseEther works
+  const defaultMonitoringReward = parseEther('5') as UInt<32>;
+  // Always enable monitoring on mainnet
+  if (network.chainId === 1 && overwrites) {
+    overwrites = {
+      ...overwrites,
+      monitoringReward: defaultMonitoringReward,
```

Comment:
Hmm, I'm not a big fan of this.. what's the point of a `overwrites`  option if it doesn't actually overwrite some? I think on the SDK monitoring is already enabled by default, including on mainnet, so it's instead being disabled by the CLI, and I think this fix would need to go there, not here.